src/python/WMCore/RequestManager/RequestDB/Settings/RequestStatus.py

```diff
@@ -35,19 +30,14 @@
 
 NextStatus = {
  "new" : ["new", "testing-approved", "assignment-approved", "rejected"],
- "testing-approved" : ["testing-approved", "testing", "test-failed", "aborted"],
- "testing" : ["testing", "tested", "test-failed", "aborted"],
- "tested" : ["tested", "assignment-approved","failed", "rejected", "aborted"],
- "test-failed" : ["test-failed", "testing-approved", "rejected", "aborted"],
  "assignment-approved" : ["assignment-approved", "assigned", "rejected"],
  "assigned" : ["assigned","acquired", "aborted", "failed"],
- "acquired" : ["running-open", "failed", "completed", "acquired", "aborted"],
- "running" : ["completed", "aborted", "failed", "epic-FAILED"],
- "running-open" : ["running-closed", "aborted", "failed", "epic-FAILED"],
- "running-closed" : ["force-complete", "completed", "aborted", "failed", "epic-FAILED"],
+ "acquired" : ["running-open", "completed", "acquired", "aborted"],
```

Comment:
Hmmm, I'm not really sure whether this is a legit use case... Can't an "acquired" workflow move to "failed" according to its input or some other constraint?